packages/react/src/create-strict-api/__tests__/index.test.tsx

```diff
@@ -0,0 +1,310 @@
+/** @jsxImportSource @compiled/react */
```

Comment:
Try to take close care when looking at this file, look at the type violations and see if it should (or shouldn't!) be one.